viewer/vue-client/src/resources/json/combinedTemplates.json

```diff
@@ -2553,8 +2528,7 @@
               "@type": "Topic",
               "inScheme": {
                 "@type": "ConceptScheme",
-                "@id": "https://id.kb.se/term/lcsh",
-                "code": "lcsh"
+                "@id": "https://id.kb.se/term/lcsh"
               },
               "prefLabel": "",
               "@id": "http://id.loc.gov/authorities/label/LCSH"
```

Comment:
I think this @id` is wrong and should be removed.

Reply:
Agreed!